src/LogViewer/index.js

```diff
@@ -275,23 +276,25 @@ export default class LogViewer extends React.Component {
   }
 
   jumpToLine(lineNumber) {
-    const { minLineHeight, offset } = this.state;
+    const { offset, lineHeight } = this.state;
 
-    window.scrollTo(0, (lineNumber - offset) * minLineHeight + minLineHeight);
+    window.scrollTo(0, (lineNumber - offset) * lineHeight + 35);
     this.setState({ didLineJump: true });
   }
 
   jumpToQueriedLine(lineNumber) {
     const interval = setInterval(() => {
-      if (this.state.minLineHeight) {
+      if (this.state.chunkHeights.length) {
         this.jumpToLine(lineNumber);
         clearInterval(interval);
       }
     }, 1000);
   }
 
   render() {
-    const { isLoading, error, toolbarOpen, showLineNumbers, wrapLines, followLog } = this.state;
+    const { isLoading, error, toolbarOpen, showLineNumbers, wrapLines, followLog, lineHeight, customStyle } = this.state;
+    const lineStyle = `#log p{min-height:${lineHeight}px}#log{line-height:${lineHeight}px}`;
+    const helmetStyle = customStyle ? customStyle + lineStyle : lineStyle;
```

Comment:
Small nit: if you rename this to `cssText`, you can save some space in the Helmet tag.